google_cdn_backend_bucket/main.tf

```diff
@@ -18,6 +18,9 @@ resource "google_compute_backend_bucket" "default" {
     max_ttl           = var.cdn_policy.max_ttl
     negative_caching  = var.cdn_policy.negative_caching
     serve_while_stale = var.cdn_policy.serve_while_stale
+    bypass_cache_on_request_headers {
+      header_name = var.cdn_policy.bypass_cache_on_request_headers.header_name
+    }
```

Comment:
This isn't exactly what I want -- as written, it will set the `bypass_cache_on_request_headers` block with a default `"Cache-Control"` value for `header_name`. I don't want to set the `bypass_cache_on_request_headers` block at all if its not provided.